src/components/WebexLocalMedia/WebexLocalMedia.jsx

```diff
@@ -58,8 +59,22 @@ export default function WebexLocalMedia({
   return (
     <div ref={mediaRef} className={cssClasses} style={style}>
       {
-        /* eslint-disable-next-line jsx-a11y/media-has-caption */
-        stream ? <video ref={ref} playsInline autoPlay /> : disabledVideo
+        stream
+          ? (
+            <>
+              {
+                mediaType === 'video'
+                  && (
+                    <Banner className="my-preview">
+                      <div className="content">My Preview</div>
+                    </Banner>
+                  )
+              }
+              {/* eslint-disable-next-line jsx-a11y/media-has-caption */}
```

Comment:
what happens if we remove this eslint directive?

Reply:
ok, seems to be an accessibility issue, you can leave it as it is https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/media-has-caption.md